scripts/lib/CIME/SystemTests/system_tests_common.py

```diff
@@ -480,16 +480,13 @@ def build_phase(self, sharedlib_only=False, model_only=False):
 """
 echo Insta pass
 echo SUCCESSFUL TERMINATION > {}/cpl.log.$LID
-cp {}/scripts/tests/cpl.hi1.nc.test {}/{}.cpl.hi.0.nc.base
+cp {}/scripts/tests/cpl.hi1.nc.test {}/{}.cpl.hi.0.nc
```

Comment:
Without this change this test fails in the run phase with 
```
ERROR: copy failed: no hist files found in rundir '/Users/jedwards/projects/scratch/TESTTESTDIFF_P1.f19_g16_rx1.A.elm_gnu.20170606_074337_ml444i/run'
```
It's not clear to me why it isn't failing on other systems.